src/devices/tuya.ts

```diff
@@ -10707,6 +10707,101 @@ const definitions: Definition[] = [
         },
         whiteLabel: [{vendor: 'ELECTSMART', model: 'EST-120Z'}],
     },
+    {
+        fingerprint: [{
+            modelID: 'TS0601',
+            manufacturerName: '_TZE204_tagezcph',
+        }],
+        model: 'PRO-900Z',
+        vendor: 'ElectSmart',
+        description: 'Thermostat for electric floor',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        onEvent: tuya.onEventSetTime,
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.enum('child_lock', ea.STATE_SET, ['LOCK', 'UNLOCK']).withLabel('Child lock'),
+            e.enum('eco_mode', ea.STATE_SET, ['OFF', 'ON']).withLabel('ECO mode').withDescription('Default: Off'),
+            e.numeric('eco_temperature', ea.STATE_SET).withValueMin(5).withValueMax(30).withValueStep(1).withUnit('°C')
+                .withDescription('Max temperature in ECO mode. Default: 20'),
+            e.text('valve_state', ea.STATE).withLabel('Heating in process'),
```

Comment:
Since it's a binary (`[36, 'valve_state', tuya.valueConverter.trueFalseInvert],`), this should also be `e.binary()`